pkg/agent/sysadvisor/plugin/qosaware/resource/memory/advisor.go

```diff
@@ -161,17 +159,8 @@ func (ra *memoryResourceAdvisor) update() {
 	ra.mutex.Lock()
 	defer ra.mutex.Unlock()
 
-	// Skip update during startup
-	if time.Now().Before(ra.startTime.Add(startUpPeriod)) {
-		klog.Infof("[qosaware-memory] skip update: starting up")
-		return
-	}
-
-	// Check if essential pool info exists. Skip update if not in which case sysadvisor
-	// is ignorant of pools and containers
-	reservePoolInfo, ok := ra.metaReader.GetPoolInfo(state.PoolNameReserve)
-	if !ok || reservePoolInfo == nil {
-		klog.Warningf("[qosaware-memory] skip update: reserve pool not exist")
+	if !ra.metaReader.HasSynced() {
```

Comment:
Note: Still need a startup period to avoid inaccurate metric value during the first several metric fetcher sync periods. Implement this startup to hasSynced in meta cache.